packages/Symfony/src/Bridge/DefaultAnalyzedSymfonyApplicationContainer.php

```diff
@@ -35,14 +37,22 @@ final class DefaultAnalyzedSymfonyApplicationContainer implements AnalyzedApplic
      */
     private $containerFactory;
 
+    /**
+     * @param ParameterProvider           $parameterProvider
+     * @param SymfonyKernelParameterGuard $symfonyKernelParameterGuard
+     * @param ContainerFactory            $containerFactory
```

Comment:
There should be no duplicated for PHP code info.
The sniff is disabled for this, because it was doing to much